examples/hitl/pick_throw_vr/config/experiment/headless_server.yaml

```diff
@@ -0,0 +1,13 @@
+# @package _global_
```

Comment:
"experiment" is an odd name for a folder that conceptually contains config overrides, but Hydra is really confusing and I was looking to match the naming convention they use in their docs and examples. The name here is unrelated to my use of "experimental" in the config, btw--in the future, we'll put all config overrides in this folder, even if they're not experimental features.